megalinter/reporters/UpdatedSourcesReporter.py

```diff
@@ -76,6 +75,7 @@ def produce_report(self):
                     except Exception as exp:
                         logging.error(
                             "[Updated Sources Reporter] Failed to git push auto fixes: " + str(exp.message)
```

Comment:
@lukelloyd1985 missing `,` at the end of this line :)

Reply:
dammit :(